go.mod

```diff
@@ -4,5 +4,6 @@ go 1.16
 
 require (
 	github.com/kylelemons/godebug v1.1.0
+	github.com/stretchr/testify v1.7.0
```

Comment:
I'm dubious about pulling in a dependency for something as trivial as what we're using this for here.